odoo_backup_sh/doc/index.rst

```diff
@@ -25,19 +25,40 @@ Configuration
   server_wide_modules = web,odoo_backup_sh
   # (...)
 
+In order for desired databases to appear in a backup dashboard, you have to create settings for it.
+Open the menu ``[[ {Backups} ]] >> {Databases} >> {Databases}``. Click on the ``[{Create}]`` button.
+Fill required fields:
 
-.. todo:: Add the section when interfaces are ready.
-{Instruction how to configure the module before start to use it}
+- Database: select one of your databases
+- Auto Backup Execute Every:  type a value and select a unit of measure.
+  Example: You want the module to make new backup every 2 hours.
```

Comment:
It must allow to specify time of backup (most oftenly used configuration is once in a night)